packages/cf-style-provider/src/index.js

```diff
@@ -36,11 +38,22 @@ export const createRenderer = opts => {
     enhancers.push(monolithic());
   }
 
-  return createFelaRenderer({
+  const renderer = createFelaRenderer({
     plugins,
     enhancers,
     selectorPrefix: [usedOpts.selectorPrefix]
   });
+
+  renderer.renderFont('Cloudflare Icons', [
+    `data:application/x-font-ttf;charset=utf-8;base64,${cloudflareIcons}`
+  ]);
+
+  const spinAnimationName = renderer.renderKeyframe(() => ({
+    from: { transform: 'rotate(0deg)' },
+    to: { transform: 'rotate(359deg)' }
+  }));
+
+  return { renderer, spinAnimationName };
```

Comment:
Actually, I'm not sure why embedding is even necessary or desired. How do we share the same font file from the backbone side and Fela side without sharing the same font URL? What about caching?